### DISCUSSION TITLE
What's the `sd_notify` requivalent in openrc?

### DISCUSSION THREAD

**Discussion by tigerinus** [OpenRC/openrc#547] [Q&A]
For `systemd`, it has a mechanism to let service to tell if it's ready (versus simply runs) that it allows service to call `sd_notify` with `READY=1` string. 

See https://www.man7.org/linux/man-pages/man3/sd_notify.3.html and https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=

What's the equivalent in openrc? 

Thanks!

**Comment by N-R-K** [comment]
Recent OpenRC supports s6 style ready notification (https://github.com/OpenRC/openrc/pull/767) there's a PR open for systemd style ready notification support too (https://github.com/OpenRC/openrc/pull/788).
